scripts/strings.py

```diff
@@ -0,0 +1,32 @@
+#
+# strings.py - print the unique strings of runtime function arguments and return values.
+#
+# uftrace-option: --nest-libcall --auto-args
+#
+
+strset = set()
+
+def uftrace_entry(ctx):
+    global strset
+    if "args" in ctx:
+        args = ctx["args"]
+        for arg in args:
+            if isinstance(arg, str):
+                arg = arg.rstrip().lstrip()
+                if arg is not "":
+                    strset.add(arg)
+
+def uftrace_exit(ctx):
+    global strset
+    if "retval" in ctx:
+        ret = ctx["retval"]
+        if isinstance(ret, str):
+            ret = ret.rstrip().lstrip()
+            if ret is not "":
+                strset.add(ret)
+
+def uftrace_end():
+    global strset
+    for strval in strset:
+        print('"%s"' % strval)
+        print("---")
```

Comment:
For similarity to `strings` util, I think it's better to remove the "---".

Reply:
I added it for strings in multiple lines, for example `ps` command, as follows:
```
$ uftrace script -S scripts/strings.py --record /bin/ps
   PID TTY          TIME CMD
136005 pts/87   00:00:00 bash
160745 pts/87   00:00:00 uftrace
160746 pts/87   00:00:00 uftrace
160747 pts/87   00:00:00 ps
":      vi
State:  T (stopped)
Tgid:   110256
Ngid:   0
Pid:    110256
PPid:   109880
TracerPid:      0
Uid:    1011    10..."
---
"NSpid: 229
NSpgid: 0
NSsid:  0
Threads:        1
SigQ:   0/256000
SigPnd: 0000000000000000
ShdPnd: 00000..."
---
"VmRSS:     1788 kB
VmData:     1668 kB
VmStk:       132 kB
VmExe:       412 kB
VmLib:      5744 k..."
---
":           152 kB
VmLib:      2092 kB
VmPTE:        44 kB
VmPMD:        12 kB
VmSwap:      332 kB
Huge..."
---
...
```

Reply:
ok then